hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -38,21 +42,47 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
   };
 
   render() {
-    const {onReload, lastUpdated, isLoading} = this.props;
-    const lastUpdatedText = lastUpdated === 0 ? 'NA' :
+    const {onReload, lastUpdated, lastUpdatedOMDBDelta,lastUpdatedOMDBFull,isLoading,lastUpdatedOMDBDeltaText,lastUpdatedOMDBFullText} = this.props;
+    
+     const lastUpdatedText = lastUpdated === 0 || lastUpdated === undefined ? 'NA' :
       (
         <Tooltip
           placement='bottom' title={moment(lastUpdated).format('ll LTS')}
         >
-          {moment(lastUpdated).format('LTS')}
+          {moment(lastUpdated).format('LT')}
         </Tooltip>
       );
+
+      const omDBDeltaFullToolTip = <span>
+          {lastUpdatedOMDBFullText}: {moment(lastUpdatedOMDBFull).fromNow()}, {moment(lastUpdatedOMDBFull).format('LT')}
+          <br/>
+          {lastUpdatedOMDBDeltaText}: {moment(lastUpdatedOMDBDelta).fromNow()}, {moment(lastUpdatedOMDBDelta).format('LT')}
```

Comment:
Here, we could just use hard-coded text label right? e.g.

```suggestion
          Full update: {moment(lastUpdatedOMDBFull).fromNow()}, {moment(lastUpdatedOMDBFull).format('LT')}
          <br/>
          Delta update: {lastUpdatedOMDBDeltaText}: {moment(lastUpdatedOMDBDelta).fromNow()}, {moment(lastUpdatedOMDBDelta).format('LT')}
```

Reply:
Current Timestamp Can be from Full Snapshot or Delta Update so I have used sort method in Descending order so If we do hardcoding of variables this logic will not apply so kept everything in separate variables.

Reply:
Sorry, I don't understand why we can't hard-code the texts here. If we hard-code here the texts (as we always want to display "`Full update`" or "`Delta update`") we don't even need the `lastUpdatedOMDBDeltaText` and `lastUpdatedOMDBFullText` variables on the `overview.tsx`, as we just add the `autoReloadPanel` to the overview page. We need two parameters for the `autoReloadPanel`, the two timestamps, the delta and full update times. So with the hard-code texts here we can also remove the text variables from the `overview.tsx` which would be easier. Let me know if I misunderstood something.